pkg/sdk/testint/tags_integration_test.go

```diff
@@ -281,3 +283,915 @@ func TestInt_TagsShowByID(t *testing.T) {
 		require.Equal(t, id2, e2.ID())
 	})
 }
+
+func TestInt_TagsAssociations(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	awsBucketUrl := testenvs.GetOrSkipTest(t, testenvs.AwsExternalBucketUrl)
+	awsRoleARN := testenvs.GetOrSkipTest(t, testenvs.AwsExternalRoleArn)
+
+	tag, tagCleanup := testClientHelper().Tag.CreateTag(t)
+	t.Cleanup(tagCleanup)
+
+	tagValue := "abc"
+	tags := []sdk.TagAssociation{
+		{
+			Name:  tag.ID(),
+			Value: tagValue,
+		},
+	}
+	unsetTags := []sdk.ObjectIdentifier{
+		tag.ID(),
+	}
+
+	t.Run("TestInt_TagAssociationForAccount", func(t *testing.T) {
+		err := client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			SetTag: tags,
+		})
+		require.NoError(t, err)
+
+		returnedTagValue, err := client.SystemFunctions.GetTag(ctx, tag.ID(), testClientHelper().Ids.AccountIdentifierWithLocator(), sdk.ObjectTypeAccount)
+		require.NoError(t, err)
+		assert.Equal(t, tagValue, returnedTagValue)
+
+		err = client.Accounts.Alter(ctx, &sdk.AlterAccountOptions{
+			UnsetTag: unsetTags,
+		})
+		require.NoError(t, err)
+
+		_, err = client.SystemFunctions.GetTag(ctx, tag.ID(), testClientHelper().Ids.AccountIdentifierWithLocator(), sdk.ObjectTypeAccount)
```

Comment:
it's a rather ugly error, weren't we supposed to do something with it?

Reply:
Next PR.

Reply:
Okay, please add to the PR description then, and I am keeping it open

Reply:
It already was: `support empty tag values in GetTag` will tackle this.